src/test/java/org/cryptomator/fusecloudaccess/FuseFillDirImpl.java

```diff
@@ -0,0 +1,46 @@
+package org.cryptomator.fusecloudaccess;
+
+import jnr.ffi.Pointer;
+import ru.serce.jnrfuse.FuseFillDir;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class FuseFillDirImpl implements FuseFillDir {
```

Comment:
Can we use a mocked filler instead?

Reply:
resolved by 580e8b613afce1e69aa3085df73644e6849eecc7